ruby/vampires.rb

```diff
@@ -43,19 +39,25 @@
     puts "Probably a vampire"
   end
 
-  if allergies != "sunshine"
-    if correct_age && !dislikes_both
-        puts "Probably not a vampire"
-    elsif !correct_age && !dislikes_both
-        puts "Probably a vampire"
-    elsif !correct_age && dislikes_both
-        puts "Almost certainly a vampire"
-    elsif name == "Tu Fang"|| name == "Drake Cula"
-        puts "Definitely a vampire"
-    else
-      puts "Results inconclusive"
-    end
+final_result = ""
+if allergies != "sunshine"
+  if correct_age && (wants_garlicbread || insurance)
+    final_result = "Probably not a vampire"
```

Comment:
I like how you used a variable to return the conditional logic.  However, this can be done with `elsif`'s if you feel comfortable on how to solve both ways then feel free to move forward.  If not reach out and we can do some further code reviewing.